db/controllers/articles-controller.js

```diff
@@ -0,0 +1,13 @@
+const { readArticleByID } = require("../models/articles-model");
+
+const getArticleByID = (req, res, next) => {
+  const path = req.path;
+  const article_id = path.slice(path.lastIndexOf("/") + 1);
```

Comment:
Rather than using req.path here, you can use req.params. This will allow you to get hold of the article_id without having to do the slice. 👍